packages/flutter/lib/fix_data.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Spec file for "dart fix", see <insert URL>.
+
+# Please add new fixes to the top of the file, separated by one blank line
+# from other fixes. In a comment, include a link to the PR where the change
+# requiring the fix was made.
+
+version: 1
+transforms:
+
+  # Changes made in https://github.com/flutter/flutter/pull/41859
+  - title: 'Removed brightness'
```

Comment:
Should this be "Remove brightness"? I believe this title is used in the quick-fix menu of IDEs, so you see it before you actually perform the action.